docs/cloud/quickstart.rst

```diff
@@ -0,0 +1,140 @@
+
+1. Find your API key. From the `API keys page <https://cloud.ibm.com/iam/apikeys>`__, view or create your API key, then copy it to a secure location so you can use it for authentication.
+2. Find your Cloud Resource Name (CRN). Open the `Instances page <https://cloud.ibm.com/quantum/instances>`__ and click your instance. In the page that opens, click the icon to copy your CRN. Save it in a secure location so you can use it for authentication.
+
+
+Optionally save your credentials to disk
+-------------------------------------------
+
+
+Optionally save your credentials to disk (in the ``$HOME/.qiskit/qiskit-ibm.json`` file). If you don't save your credentials to disk, you must specify your credentials every time you start a new session.
+
+If you save your credentials to disk, you can use ``QiskitRuntimeService()`` in the future to initialize your account.
+
+.. code-block:: python
+
+   from qiskit_ibm_runtime import QiskitRuntimeService
+
+   # Save account to disk.
+   QiskitRuntimeService.save_account(channel="ibm_cloud", token="<IBM Cloud API key>", instance="<IBM Cloud CRN>")
+
+   service = QiskitRuntimeService()
+
+
+If you need to update your saved credentials, run ``save_account`` again, passing in ``overwrite=True``  and the updated credentials. For more information about managing your account, see the `account management tutorial <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/tutorials/04_account_management.html>`__.
+
+Test your setup
+-------------------
+
+Run the Hello World program to ensure that your environment is set up properly.
+
+.. note::
+   If you are using the Standard plan, running Hello World incurs a cost. See `Manage costs <cost.html>`__ for cost information.
+
+If you did not save your credentials to disk, specify ``QiskitRuntimeService(channel="ibm_cloud", token=<IBM Cloud API key>, instance=<IBM Cloud CRN>)``
+instead of ``QiskitRuntimeService()`` in the following code.
+
+.. code-block:: python
+
+   from qiskit_ibm_runtime import QiskitRuntimeService
+
+   service = QiskitRuntimeService()
+   program_inputs = {'iterations': 1}
+   options = {"backend_name": ""}
+   job = service.run(program_id="hello-world",
+                  options=options,
+                  inputs=program_inputs
+                 )
+   print(f"job id: {job.job_id}")
+   result = job.result()
+   print(result)
+
+
+Result:
+
+.. code-block::
+
+       Hello world!
+
+
+Choose a program to run
+----------------------------------
+
+
+Qiskit Runtime uses primitive programs to interface with quantum computers. The following programs are publicly available. Choose the appropriate link to continue learning how to run a program.
+
+`Getting started with Sampler <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/tutorials/how-to-getting-started-with-sampler.html>`__
+`Getting started with Estimator <https://qiskit.org/documentation/partners/qiskit_ibm_runtime/tutorials/how-to-getting-started-with-estimator.html>`__
```

Comment:
The rendered page has both links on the same line, which is hard to read. I think an empty line between them would fix it.